doc/concepts/morphology.rst

```diff
@@ -183,6 +258,8 @@ which is illustrated along with its branches below.
   :width: 800
   :align: center
 
+  Left, the same 10 segment cable cell seen before. On the right, the associated morphology and branches.
+  Note that the proximal point of the soma is always the start (and possibly end) of a branch.
```

Comment:
I don't understand what this sentence means. The proximal point of a soma does not have to be the start of a branch, unless you mean the soma branch itself?

Reply:
Indeed. Note that there is no such thing as a soma branch: the segments of the dendrite are here included in the 'first branch' up until the first branching point. The axon here is making the proximal point of the soma _also_ the distal point of another branch. What would make it more clear?

Reply:
In theory, there is nothing stopping the soma from being in the middle of a branch. I think you mean the `root` instead of the `proximal point of the soma.`

Reply:
Right!